lib/models/get_repositories/repos_result_rto.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:json_annotation/json_annotation.dart';
+
+import 'repository_rto.dart';
+
+part 'repos_result_rto.g.dart';
+
+@JsonSerializable(fieldRename: FieldRename.snake)
+class ReposResultRTO {
+  final List<RepositoryRTO> items;
+
+  ReposResultRTO({required this.items});
```

Comment:
In our Flutter project, constructors are defined first, then the variables. Could you switch the order?

Reply:
Also in other files

Reply:
done